apps/addons/templates/addons/review_add_box.html

```diff
@@ -14,6 +14,7 @@
     <form disabled method="post" action="{{ url('addons.reviews.add', addon.slug) }}">
       {% set attrs = {} if user.is_authenticated() else {'disabled': 'disabled'} %}
       {{ csrf() }}
+      {{ pretty_field(form.title, label=_('Title')) }}
```

Comment:
isn't it `review_form`? It seems from the failing test that this code is causing issues when reporting an abuse.